kubernetes/pipelines/deploy_core/Jenkinsfile

```diff
@@ -2,8 +2,8 @@
 node() {
     try {
         stage('checkout public repo') {
-            cleanWs()
             checkout scm
+            sh'git clean -fxd'
```

Comment:
Not exactly sure if we need to remove cleanws. This we will take it as an experiment and see how it goes.

Reply:
cleanWs() is removed because always clone is taking more time. It makes more sense to pull the delta and do a git clean to remove previous templating or other changes.